.circleci/deploy-to-pantheon.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Is any of the logic in here WordPress specific? If not, I think we should encapsulate into the build plugin itself.